apps/gitness/src/layouts/PullRequestLayout.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { NavLink, Outlet } from 'react-router-dom'
+import { Badge, Icon, Spacer, Tabs, TabsList, TabsTrigger } from '@harnessio/canary'
+import { Floating1ColumnLayout, PullRequestHeader } from '@harnessio/playground'
+// import { useGetPullReqQuery } from '@harnessio/code-service-client'
+
+const mockedPullRequest = {
```

Comment:
Any reason we're still using mock data here?

Reply:
API was stuck in infinite loop, need to fix this Layout